botfront/imports/lib/utils.js

```diff
@@ -80,15 +80,6 @@ export const getProjectModelLocalFolder = () => process.env.MODELS_LOCAL_PATH ||
 
 export const getProjectModelLocalPath = projectId => path.join(getProjectModelLocalFolder(), getProjectModelFileName(projectId, 'tar.gz'));
 
-export function uploadFileToGcs(filePath, bucket) {
-    const { Storage } = require('@google-cloud/storage');
```

Comment:
Make sure to uninstall this.

Reply:
moved to https://github.com/botfront/botfront/pull/856 so I don't have to re-reun the jobs on this branch and potentially delay the release

Reply:
Oops

Reply:
no problem, better to do it here really 🙂